pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -588,10 +582,58 @@ func convertToUsers(guestInfo *libvirt.DomainGuestInfo) []api.User {
 	return users
 }
 
+func findDisk(guestInfoDisks []libvirt.DomainGuestInfoDisk, serial string) libvirt.DomainGuestInfoDisk {
+	for _, disk := range guestInfoDisks {
+		if disk.Serial == serial {
+			return disk
+		}
+	}
+	// Unreachable
```

Comment:
Does "unreachable" means it's a dead-code here? If so, in that case, just have one single return in the end that will return an existing or empty disk.